lib/tapioca/commands/abstract_gem.rb

```diff
@@ -74,15 +77,22 @@ def initialize(
       def gems_to_generate(gem_names)
         return @bundle.dependencies if gem_names.empty?
 
-        gem_names.map do |gem_name|
+        gem_candidates(gem_names).map do |gem_name|
```

Comment:
No strong opinions but I prefer being explicit in the call site to have easier readability in the future.

```ruby
gem_names = gems_and_their_dependencies(gem_names) if @include_dependencies
gem_names.map do |gem_name|
  ...
end
```